task_test.go

```diff
@@ -594,6 +595,36 @@ func (s *TaskSuite) TestImagePlatform() {
 	s.Equal("arm64", configFile.Architecture)
 }
 
+func (s *TaskSuite) TestOciImage() {
+	s.req.Config.ContextDir = "testdata/multi-arch"
+	s.req.Config.ImagePlatform = "linux/arm64,linux/amd64"
+	s.req.Config.OutputOCI = true
+
+	_, err := s.build()
+	s.NoError(err)
+
+	l, err := layout.ImageIndexFromPath(s.imagePath("image"))
+	s.NoError(err)
+
+	im, err := l.IndexManifest()
+	s.NoError(err)
+
+	desc := im.Manifests[0]
+	ii, err := l.ImageIndex(desc.Digest)
+	s.NoError(err)
+
+	images, err := ii.IndexManifest()
+	s.NoError(err)
```

Comment:
@xtremerui unfortunately it is, I was very confused by this at first as well. The first IndexManifest is for the bundle it self (the directory containing the files), you then provide the "Digest" of that bundle to get it's index (json file kept inside the directory) and then you retrieve that manifest using IndexManifest and that's where the data for all the images (multiple images because of multiple CPU architectures) is contained.

When I was figuring this out I actually added print statements to the test, here's a sample output from that so you can see how they differ:
```
Contents of im: &{SchemaVersion:2 MediaType: Manifests:[{MediaType:application/vnd.oci.image.index.v1+json Size:703 Digest:sha256:1e0d02719eea72a55306e7d55d9d84b6ac25c3ac5705e0d78d72f26eee18e649 Data:[] URLs:[] Annotations:map[org.opencontainers.image.created:2022-07-13T16:48:31Z] Platform:<nil>}] Annotations:map[]} 

Contents of images: &{SchemaVersion:2 MediaType:application/vnd.oci.image.index.v1+json Manifests:[{MediaType:application/vnd.oci.image.manifest.v1+json Size:710 Digest:sha256:a6f9c6d46f97a85ace889664bc42437cff0e59c769b02629d98470378ff659d0 Data:[] URLs:[] Annotations:map[] Platform:linux/arm64} {MediaType:application/vnd.oci.image.manifest.v1+json Size:710 Digest:sha256:84b4c953792a04908939367e04c74baa03d8e4f2e701b1c3aa20751e9d286089 Data:[] URLs:[] Annotations:map[] Platform:linux/amd64}] Annotations:map[]} 
```